examples/map_to_physical_robot/README.md

```diff
@@ -0,0 +1,10 @@
+### Running the Robot without the RemoteController
+* To manually run the physical robot you first have to create a config pickle file as shown in `make_config_file`.
+* Copy the config file onto the physical robot using: `scp <config file path> <user>@<rpi-ip>:<desired path>`. (Keep in mind that for this you have to have a properly setup raspberry pi controller on the robot. If you dont -> check the documentation for a tutorial.)
```

Comment:
your stuff between brackets is very important requirement and should be at the start of this example